### PR TITLE
SM-634: Fix file extension bug on export

### PR DESCRIPTION
**Pull request by coltonhurst**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

My recent [PR](https://github.com/bitwarden/clients/pull/4944) accidentally changed the file extension during export download. This PR fixes that bug.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <bit-form-field class="tw-max-w-sm">
     <bit-label>{{ "fileFormat" | i18n }}</bit-label>
     <select bitInput formControlName="format">
-      <option *ngFor="let format of exportFormats" [ngValue]="format">{{ format }}</option>
+      <option *ngFor="let format of exportFormats">{{ format.name }}</option>
```
Comment (Hinton) [inline bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.html:13]: We could use the index here instead which simplifies the logic.

```suggestion
      <option *ngFor="let format of exportFormats; let i = index" [value]="index">{{ format.name }}</option>
```

File: bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.ts [hunk 2]
```diff
@@ -76,12 +81,13 @@ export class SecretsManagerExportComponent implements OnInit, OnDestroy {
   };
 
   private async doExport() {
-    const exportData = await this.secretsManagerApiService.export(
-      this.orgId,
-      this.formGroup.get("format").value
-    );
+    const fileExtension = this.exportFormats.filter(
+      (format) => format.name == this.formGroup.get("format").value
+    )[0].fileExtension;
```
[the remark this answers is not in the document]
Comment (coltonhurst) [reply bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.ts]: That's a great point, thank you for bringing that up! We removed the need for any search based on the new way @Hinton mentioned above, but next time with a similar case I'll use find 😁

File: bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.html (hunk 1 above)
Comment (coltonhurst) [reply bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.html]: Updated

**Review by Hinton — APPROVED** [review]
LGTM

**Pull request merged**